submissions/Nik3264/js-dom/style.css

```diff
@@ -0,0 +1,172 @@
+* {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+  font-family: "Oswald", sans-serif;
+}
+
+:root {
+  --color-header: #8d7361;
+  --color-back: #eeebe7;
+  --color-text: #50423d;
+  --color-blue: #8b8da0;
+  --color-yellow: #c9bbae;
+}
+
+@keyframes fade {
```

Comment:
I'm stuck on this animation. It would be easier to remove it immediately. And spend more time on the code)))
To be honest, I like learning js more than css.

Reply:
I forgot about this one. Change `translateX(100px);` to `translateY(-25px);` and `opacity: 0;` to `opacity: 0.75;` Don't forget about 100% step of animation there `translateX` replace to 'translateY`